src/posts/what-is-wai.md

```diff
@@ -0,0 +1,108 @@
+---
+title: What is WAI? The Web Accessibility Initiative
+description: "The Web Accessibility Initiative is W3C's effort to improve accessibility of the Web for people with disabilities"
+category: concepts
+author: Ilknur Eren
+date: 2022-07-18
+tags:
+  - concepts
+---
+
```

Comment:
Can we have a very high-level sentence or two describing what we'll be covering in this post? You could even repeat what you wrote for the background metadata.